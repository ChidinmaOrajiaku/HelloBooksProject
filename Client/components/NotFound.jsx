import React from 'react';
import BodyNavigationBar from './BodyNavigationBar';
import Footer from './Footer';
import notFound from '../img/notfound.gif';

export default () => (
  <div>
    <div className=""> <BodyNavigationBar /> </div>
    <div className="notFound">
      <div className="container">
        <div className="card">
          <h1>404</h1>
          <img src={notFound}/>
          <h2>Page Not Found</h2>
        </div>
      </div>
      <div> <Footer /></div>
    </div>
  </div>
);
