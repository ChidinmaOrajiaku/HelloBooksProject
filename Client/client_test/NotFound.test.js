import React from 'react';
import $ from 'jquery';
import toJson from 'enzyme-to-json';
import { configure, shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import  NotFound from '../components/NotFound.jsx';

configure({ adapter: new Adapter() });

describe('<NotFound />', () => {
  it('should render component without breaking', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toBeDefined();
  });
  it('should match snapshot test', () => {
    const component = shallow(<NotFound />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});