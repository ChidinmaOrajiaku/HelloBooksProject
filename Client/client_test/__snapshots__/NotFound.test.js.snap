// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should match snapshot test 1`] = `
<div>
  <div
    className=""
  >
     
    <Component />
     
  </div>
  <div
    className="notFound"
  >
    <div
      className="container"
    >
      <div
        className="card"
      >
        <h1>
          404
        </h1>
        <img
          src={Object {}}
        />
        <h2>
          Page Not Found
        </h2>
      </div>
    </div>
    <div>
       
      <Component />
    </div>
  </div>
</div>
`;
